### PR TITLE
Truncate git commit messages to recommended 50

### LINKED ISSUE #1: Truncate the commit message if it's past a certain length

### PR DESCRIPTION
**Pull request by praveenperera**
Closes #1

### CODE REVIEW

File: app/views/shared/_commit_list.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <ul class="commits">
   <% commits.each do |commit| %>
     <li class="commit">
-      <strong><%= commit.message %> <a href="<%= "#{commit.github_url}" %>" class="notice"><%= commit.formatted_sha1 %></a></strong>
+      <strong><%= truncate(commit.message, length: 50) %> <a href="<%= "#{commit.github_url}" %>" class="notice"><%= commit.formatted_sha1 %></a></strong>
```
Comment (vipulnsward) [inline app/views/shared/_commit_list.html.erb]: @praveenperera where was the recommended 50 picked up? Think it should be 80, or GitHub's 118.

**Comment by praveenperera** [comment]
50/72 is the what I've seen online the most. I also follow to follow this article for my commits:

https://chris.beams.io/posts/git-commit/

Also GitHub UI seems to also suggest 50. 
Screen Shot 2019-06-19 at 2 50 00 PM

**Comment by vipulnsward** [comment]
Thanks @praveenperera !

**Comment by slavingia** [comment]
very nice! thanks!

**Pull request merged**